tensorboard/manager_test.py

```diff
@@ -71,11 +71,12 @@ def test_roundtrip_serialization(self):
     self.assertEqual(also_info, info)
 
   def test_serialization_rejects_bad_types(self):
-    info = _make_info()._replace(start_time=1549061116)  # not a datetime
+    bad_time = datetime.datetime.fromtimestamp(1549061116)  # not an int
+    info = _make_info()._replace(start_time=bad_time)
     with six.assertRaisesRegex(
         self,
         ValueError,
-        "expected 'start_time' of type.*datetime.*, but found: 1549061116"):
+        "expected 'start_time' of type.*int.*, but found: datetime\."):
```

Comment:
noob question: why do you need a leading blackslash in front of the "."? It does not look like a regex or it requires any escaping.

Reply:
It’s a regex: note `with six.assertRaisesRegex`, and note the
metacharacters earlier in the string (`type.*int.*`).